platform/macos/export/export_plugin.cpp

```diff
@@ -1264,7 +1264,7 @@ void EditorExportPlatformMacOS::_code_sign_directory(const Ref<EditorExportPrese
 			continue;
 		}
 
-		if (extensions_to_sign.has(current_file.get_extension())) {
+		if (extensions_to_sign.has(current_file.get_extension().to_lower())) {
```

Comment:
I'd leave these separately, they're unrelated and does nominally change behavior, so would be better to not bundle them with a new feature